container/shim/src/modules/log_ingestor.js

```diff
@@ -112,17 +112,21 @@ function parseSingleLine(line) {
  */
 async function submitBandwidthLogs(logs) {
   // calculate total bytes sent to clients and cache hit rate for this batch or retrievals
-  const { bytesSent, cacheHits } = logs.reduce(
+  const { bytesSent, validReqs, cacheHits } = logs.reduce(
     (acc, log) => ({
       bytesSent: acc.bytesSent + log.numBytesSent, // sum of all bytes sent to clients
+      validReqs: acc.validReqs + (log.httpStatusCode === 200 ? 1 : 0), // sum of all valid hits
       cacheHits: acc.cacheHits + (log.cacheHit ? 1 : 0), // sum of all cache hits
```

Comment:
Should also filter for 200 status for `cacheHits`. We can have cacheHits for any status code.